<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pad_motion_settings_dialog</class>
 <widget class="QDialog" name="pad_motion_settings_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Motion Controls</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_main">
   <item>
    <widget class="QGroupBox" name="gb_device">
     <property name="title">
      <string>Device</string>
     </property>
     <layout class="QHBoxLayout" name="gb_device_layout">
      <item>
       <widget class="QComboBox" name="cb_choose_device"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_axis_x">
     <property name="title">
      <string>X</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_x" stretch="2,1,1,0,1">
      <item>
       <widget class="QSlider" name="slider_x">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_x">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_x"/>
      </item>
      <item>
       <widget class="QCheckBox" name="mirror_x">
        <property name="text">
         <string>Mirrored</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="shift_x">
        <property name="prefix">
         <string>Shift: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_axis_y">
     <property name="title">
      <string>Y</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_y" stretch="2,1,1,0,1">
      <item>
       <widget class="QSlider" name="slider_y">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_y">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_y"/>
      </item>
      <item>
       <widget class="QCheckBox" name="mirror_y">
        <property name="text">
         <string>Mirrored</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="shift_y">
        <property name="prefix">
         <string>Shift: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_axis_z">
     <property name="title">
      <string>Z</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_z" stretch="2,1,1,0,1">
      <item>
       <widget class="QSlider" name="slider_z">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_z">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_z"/>
      </item>
      <item>
       <widget class="QCheckBox" name="mirror_z">
        <property name="text">
         <string>Mirrored</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="shift_z">
        <property name="prefix">
         <string>Shift: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_axis_g">
     <property name="title">
      <string>G</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_g" stretch="2,1,1,0,1">
      <item>
       <widget class="QSlider" name="slider_g">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_g">
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_g"/>
      </item>
      <item>
       <widget class="QCheckBox" name="mirror_g">
        <property name="text">
         <string>Mirrored</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="shift_g">
        <property name="prefix">
         <string>Shift: </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="main_vertical_spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_box_layout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="button_box_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>pad_motion_settings_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>pad_motion_settings_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
